miden-rpc/src/config.rs

```diff
@@ -0,0 +1,55 @@
+use std::fmt::Display;
+
+use miden_store;
+use serde::{Deserialize, Serialize};
+
+pub const HOST: &str = "localhost";
+// defined as: sum(ord(c)**p for (p, c) in enumerate('miden-rpc', 1)) % 2**16
+pub const PORT: u16 = 57291;
+pub const ENV_PREFIX: &str = "MIDEN_RPC";
+pub const CONFIG_FILENAME: &str = "miden-rpc.toml";
+
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
+pub struct Endpoint {
+    /// The host device the server will bind to.
+    pub host: String,
+    /// The port number to bind the server to.
+    pub port: u16,
+}
+
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
+pub struct RPCConfig {
+    /// Main endpoint of the server.
+    pub endpoint: Endpoint,
+    /// Address of the database server in the format `http://<host>[:<port>]`.
+    pub db: String,
```

Comment:
nit: should we call this `store` rather than `db`?

Reply:
done